go.mod

```diff
@@ -10,6 +10,8 @@ require (
 	github.com/google/go-github v17.0.0+incompatible
 	github.com/hashicorp/go-multierror v0.0.0-20171204182908-b7773ae21874
 	github.com/kr/pretty v0.1.0
+	github.com/onsi/ginkgo v1.7.0 // indirect
+	github.com/onsi/gomega v1.4.3 // indirect
```

Comment:
not sure why this is added, I just removed this?

Reply:
When I do "go mod tidy", these show up.

Reply:
huh.. when I do `go mod tidy` these go away.. what version of go do you have?

Reply:
1.12.8

Reply:
I am on 1.13, I guess that is why. the build tools image settles on adding this, so I suppose that is the right move